control/server.py

```diff
@@ -180,6 +200,8 @@ def _start_spdk(self):
             f" conn_retries: {conn_retries}, timeout: {timeout}",
         })
         try:
+            wait_for_condition(lambda: os.path.exists(spdk_rpc_socket), f"file '{spdk_rpc_socket}' not found", timeout)
```

Comment:
As I said previously, I don't like very much we are duplicating the spdk client code here, while our implementation is less reliable (can not detect a dead socket) and potentially adds 1 sec delay on start (due to 1 sec check interval). I see you have added the socket file cleanup on termination, but it still may not work for all cases, like the host crash.

Actually I don't mind to have this, though personally I would prefer to rely on the spdk for this stuff, i.e. I would just decrease the `conn_retries`. I think I would even remove `conn_retries` from the config, and just calculated it something like this (assuming the spdk client retries every 0.2 sec): `conn_retries = min(timeout / 0.2, 10)`.

Reply:
sorry, ofc I meant  `conn_retries = max(timeout / 0.2, 10)` here.